roles/docker/tasks/cleanup.yml

```diff
@@ -12,3 +12,9 @@
   become: true
   with_items:
     - "{{ docker_api_port }}"
+
+- name: cleanup contiv-compose
```

Comment:
At first glance, I see two problems:
1) Obvious... contiv-compose is checked against docker-compose version instead of contiv-compose.
2) Per ansible documentation http://docs.ansible.com/ansible/shell_module.html, the creates clause will cause the task to be skipped if the file exists.

Hope this helps.